pom.xml

```diff
@@ -165,6 +165,14 @@
         </repository>
       </repositories>
     </profile>
+
+    <profile>
+      <id>jdk11</id>
+      <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
```

Comment:
@JimClarke5 , can you add `maven.compiler.release=11` as well, as suggested by @Craigacp ? Then we'll be good to merge it

Reply:
Just pushed it.